esi_leap/db/sqlalchemy/api.py

```diff
@@ -287,7 +287,6 @@ def lease_get_all(filters):
 
     start = filters.pop('start_time', None)
     end = filters.pop('end_time', None)
-    owner = filters.pop('owner', None)
```

Comment:
I don't understand why owner has been removed in filters here. Looks like the Lease data model has a new lease_owner_id_idx index but it's not been used?

Reply:
What this code does is actually remove the special filters - for example start and end - and use special processing for those specific filters that you can see in the code after this line. The fields that are *not* popped are used as a simple equality filter. This line is removed because now we no longer have to use special filtering processing (joining with an offer) for an owner.

Reply:
my brilliant code at work

Reply:
I see!